pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "nautobot-secrets-providers"
-version = "1.4.1b1"
-description = "Nautobot Secrets Providers Plugin."
-authors = ["Network to Code, LLC <info@networktocode.com>"]
+version = "2.0.0rc1"
```

Comment:
```suggestion
version = "2.0.0"
```

Reply:
This will probably need a lockfile update

Reply:
Didnt really need one it seems